main.css

```diff
@@ -0,0 +1,41 @@
+body {
+  background-color: #efefef;
+}
+
+.container {
+  position: absolute;
```

Comment:
Once the SVG logo is set as the background, you won't need to absolutely position this.